.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
+    "dot-notation": 2
```

Comment:
Tom likes things as strict as possible, and we might as well keep things consistent when possible.